### PR TITLE
UI for prop cancellation

### PR DESCRIPTION
**Pull request by EricDevito**
- Only available to proposer during pending/active/queued proposal states

### CODE REVIEW

File: packages/nouns-webapp/src/pages/Vote/Vote.module.css [hunk 1]
```diff
@@ -71,6 +71,33 @@
...
+  background-color: var(--brand-color-red);
+}
+
+.desturctiveTransitionStateButton:hover {
+  background-color: var(--brand-color-red);
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+.desturctiveTransitionStateButtonSection {
```
Comment (solimander) [inline packages/nouns-webapp/src/pages/Vote/Vote.module.css:91]: `desturctive` -> `destructive`

File: packages/nouns-webapp/src/pages/Vote/index.tsx [hunk 2]
```diff
@@ -134,6 +152,17 @@ const VotePage = ({
     };
   })();
 
+  const desctructiveStateButtonAction = isCancellable ? <Trans>Cancel</Trans> : '';
+  const desctructiveStateAction = (() => {
```
Comment (solimander) [inline packages/nouns-webapp/src/pages/Vote/index.tsx:156]: `desctructive` -> `destructive`

**Comment by solimander** [comment]
The button text is not rendering correctly. Presumably because it's inside a `Trans` tag and there is no accompanying translation.

text

File: packages/nouns-webapp/src/pages/Vote/Vote.module.css (hunk 1 above)
Comment (EricDevito) [reply packages/nouns-webapp/src/pages/Vote/Vote.module.css]: corrected in commit
https://github.com/EricDevito/nouns-monorepo/commit/add31e1d8bf70e4f2b9f0a5b820d60fa3e0a0d31
https://github.com/EricDevito/nouns-monorepo/commit/493711d4c9607714c3f3d538f457cd38a7bb169e

File: packages/nouns-webapp/src/pages/Vote/index.tsx (hunk 2 above)
Comment (EricDevito) [reply packages/nouns-webapp/src/pages/Vote/index.tsx]: corrected in commit
https://github.com/EricDevito/nouns-monorepo/commit/493711d4c9607714c3f3d538f457cd38a7bb169e

**Comment by EricDevito** [comment]
> The button text is not rendering correctly. Presumably because it's inside a `Trans` tag and there is no accompanying translation.
> 
> text

addressed translation issues via `yarn i18n:extract` https://github.com/EricDevito/nouns-monorepo/commit/e121951f05ac2af18f7251f5de75a0cc1cbb99a0

**Review by solimander — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**